docs/source/ledger-nano.rst

```diff
@@ -0,0 +1,261 @@
+.. _ledger-nano:
+
+Manage XNS via Ledger Nano hardware wallet
+==========================================
+
+`Ledger Nano S <https://shop.ledger.com/products/ledger-nano-s>`_ and `Nano X <https://shop.ledger.com/products/ledger-nano-x>`_ are hardware wallets built by `Ledger <https://www.ledger.com/>`_.
+
+A hardware wallet is a separate device that stores the private keys for your crypto assets. The keys never leave the device and remain secure even if the device is connected to a compromised computer.
+
+Insolar application for Ledger Nano S and X keeps your Insolar private key on the device and lets you manage XNS coins via the `Insolar Wallet <https://wallet.insolar.io>`_.
+
+The application is developed by Insolar team. To request technical support, send an email to support@insolar.io.
+
+Prerequisites
+-------------
+
+To use the Insolar application on your Ledger Nano S or X hardware wallet, make sure to install and set up the following:
+
+#. `Install Google Chrome <https://www.google.com/chrome/>`_. It's the only supported browser for now.
+#. `Install the Ledger Live application <https://support.ledger.com/hc/en-us/articles/360006395553/>`_. It allows you to manage applications on your device.
+#. If you don't yet have an Insolar Wallet, set up your `Nano S <https://support.ledger.com/hc/en-us/articles/360000613793>`_ or `Nano X <https://support.ledger.com/hc/en-us/articles/360018784134>`_.
+
+   .. important::
+
+     During the setup, you are required to choose a PIN and write down the recovery phrase. Without the PIN you will not be able to unlock the device and without the recovery phrase—restore access to your wallet.
```

Comment:
Without the PIN you will not be able to unlock the device and without the recovery phrase **you will be unable** restore access to your wallet.

Reply:
_Also, here the recovery phrase is one for Ledger? Or is it the Insolar one? Just that below we talk about backup phrase_

Reply:
you will be unable **to** restore?

Reply:
correct, good spot